examples/medical-center/generated/persist_client.bal

```diff
@@ -3,22 +3,22 @@
 // This file is an auto-generated file by Ballerina persistence layer for medical_center.
 // It should not be modified by hand.
 
-import ballerinax/mysql;
 import ballerina/persist;
 import ballerina/jballerina.java;
+import ballerinax/mysql;
 
-const MEDICAL_ITEM = "medicalitem";
-const MEDICAL_NEED = "medicalneed";
+const MEDICAL_ITEM = "medicalitems";
+const MEDICAL_NEED = "medicalneeds";
 
-public client class MedicalCenterClient {
+public client class Client {
     *persist:AbstractPersistClient;
 
     private final mysql:Client dbClient;
 
     private final map<persist:SQLClient> persistClients;
 
     private final record {|persist:Metadata...;|} metadata = {
-        "medicalitem": {
+        "medicalitems": {
```

Comment:
```suggestion
        [MEDICAL_ITEM]: {
```

Reply:
Fixed in [d0b65f0](https://github.com/ballerina-platform/module-ballerina-persist/pull/177/commits/d0b65f00336f1917043ae3b82e35b0c405656d9f)